docs/guides/functions.ipynb

```diff
@@ -163,7 +163,9 @@
     "- **`QUEUED`**: The remote program is in the Qiskit Function queue. The queue priority is based on how much you've used Qiskit Functions.\n",
     "- **`INITIALIZING`**: The remote program is starting; this includes setting up the remote environment and installing dependencies.\n",
     "- **`RUNNING`**: The program is running.\n",
-    "- **`DONE`**: The program is complete, and you can retrieve data stored in `save_result()` with `job.results()`."
+    "- **`DONE`**: The program is complete, and you can retrieve data stored in `save_result()` with `job.results()`.\n",
```

Comment:
```suggestion
    "- **`DONE`**: The program is complete, and you can retrieve result data with `job.results()`.\n",
```

End users wouldn't know what `save_result()` is, since it's only used inside a Qiskit Function's code.